usage_test.go

```diff
@@ -309,3 +309,42 @@ Global options:
 	p.WriteHelp(&help)
 	assert.Equal(t, expectedHelp, help.String())
 }
+
+func TestUsageWithOptionalLongNames(t *testing.T) {
```

Comment:
can you add an additional test that includes one options with a long name and then an option with a short name only, in order to test the ordering of short-only before long?